src/datagrid/DataGrid.stories.tsx

```diff
@@ -420,12 +420,12 @@ storiesOf("DataGrid", module)
             />
         </div>
     ))
-    .add("Grid with pagination and default pageSizes dropdown", () => (
+    .add("Grid with pagination and pageSizes dropdown", () => (
```

Comment:
Can we change this to "Grid with pagination and custom page size" instead as other existing stories also has pagesizes dropdown implemented but those stories doesn't support custom page size?

Reply:
done